### PR TITLE
Feature get supported tokens

### LINKED ISSUE #1: Get supported tokens

### LINKED ISSUE #2: Get exchange rates

### PR DESCRIPTION
**Pull request by surajsingla333**
Developed `getSupportedTokens()` to get the supported tokens available for swap

### CODE REVIEW

**Comment by surajsingla333** [comment]
Added `getExchangeRate()` to get the exchange rates.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/getsafle/1inch-controller.git"
+  },
+  "author": "getsafle",
+  "contributors": [
```
Comment (sshubhamagg) [inline package.json]: Update all contributors @surajsingla333

**Review by sshubhamagg — COMMENTED** [review]
Please update the Readme @surajsingla333

**Review by sshubhamagg — APPROVED** [review]

**Review by sshubhamagg — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "1inch-controller",
```
Comment (ARJUN-R34) [inline package.json]: @surajsingla333  Add the org name before the repo name.

@getsafle/1inch-controller

**Review by ARJUN-R34 — APPROVED** [review]

**Review by ARJUN-R34 — APPROVED** [review]

**Pull request merged**